src/config/schema.types.d.ts

```diff
@@ -16,6 +16,12 @@ type AccountLimitsConfig = {
   }
 }
 
+type DynamicField = {
```

Comment:
```suggestion
type CustomField = {
```

Reply:
this would be my suggestion regarding name

Reply:
done https://github.com/GaloyMoney/galoy/pull/1465/commits/c536cd781df7aba2f2611bb8675e4ac38188f347#diff-f10c7a06ab8f35c61100a7958d70ddfba331341e82be0017d9c7b955acdc230fR19